cmake/OpenVDBHoudiniSetup.cmake

```diff
@@ -299,6 +299,25 @@ if(NOT ILMBASE_LIBRARYDIR)
   set(ILMBASE_LIBRARYDIR ${_HOUDINI_LIB_DIR})
 endif()
 
+# OpenVDB
+
+if(NOT OPENVDB_INCLUDEDIR)
+  set(OPENVDB_INCLUDEDIR ${_HOUDINI_INCLUDE_DIR})
+endif()
+if(NOT OPENVDB_SESI_LIB)
+  # Full path to openvdb_sesi library
+  find_library(OPENVDB_SESI_LIB openvdb_sesi
+    ${_FIND_HOUDINI_ADDITIONAL_OPTIONS}
+    PATHS ${_HOUDINI_LIB_DIR}
+  )
+  if(NOT EXISTS ${OPENVDB_SESI_LIB})
+    message(WARNING "The OpenVDB Houdini CMake setup is unable to locate libopenvdb_sesi "
+      "within the Houdini installation at: ${_HOUDINI_LIB_DIR}. "
+      "This is needed for building the ABI tests."
+    )
+  endif()
+endif()
```

Comment:
This should probably be specific for the ABI tests? Otherwise I can see confusion with the value of `OPENVDB_INCLUDEDIR`. Perhaps just allow the values of `_HOUDINI_INCLUDE_DIR` and `_HOUDINI_LIB_DIR` to persist once `OpenVDBHoudiniSetup` has been called?

Also, I'd try initialising the openvdb_sesi library with `_houdini_create_libraries`

Reply:
Yeah, that's fair, I can lift this into the CMake for the abi test itself. I'm a bit confused about your comment regarding `_houdini_create_libraries` though - doesn't this get called by `OpenVDBHoudiniSetup.cmake`?

Reply:
I've moved the logic regarding the openvdb include and libraries into the CMake for the ABI test now.